### ISSUE TITLE
End-to-End Example for Running a tiny Minio in a Laptop (Minikube) for Developers Missing

### ISSUE THREAD

**Issue by bigbitbus** [minio/operator#580]
I am a developer and simply want to use minio for testing my S3 connectivity and API. I cant find a good way of running the tiny Minio I need for just functional testing.
## Expected Behavior

Minio was supposed to be developer friendly, but this operator has no focus on the developer.

## Current Behavior

It seems to assume everyone is running minio on a large datacenter/cloud setting.

## Possible Solution

Include an end-to-end installation of minio in a very constrained K8s cluster like Minikube.

## Context

I am looking for another object storage solution that can easily be installed inside a developer's laptop running Minikube now.

## Your Environment

* Version used (`minio-operator`):
* Environment name and version (e.g. kubernetes v1.17.2):
* Server type and version:
* Operating System and version (`uname -a`):
* Link to your deployment file:

Developer laptop.

**Comment by nitisht** [comment]
Please try 

```
   kubectl krew update
   kubectl krew install minio
   kubectl minio init
   kubectl minio tenant create minio-tenant-1 \
      --servers 4                             \
      --volumes 16                            \
      --capacity 16Ti                         \
```

on your Minikube and let us know what is not working.

**Comment by bigbitbus** [comment]
Will those numbers - 4 servers, 16 volumes and 16TB be appropriate for a laptop?

**Comment by dnskr** [comment]
Minimal config which works on my laptop:
- servers 1
- volumes 4
- capacity 10Gi

It would be helpful to have tiny-tenant in [examples ](https://github.com/minio/operator/tree/master/examples) and helm chart documentation.

**Comment by bigbitbus** [comment]
Thanks!

I just regressed to creating a "pod" (from a yaml file) with very small numbers. Breaks the goodness of having the exact same Infra-as-code in dev versus production, but I didnt want to deal with the complexity of figuring out the laptop-appropriate settings in the operator myself.

**Comment by nitisht** [comment]
Fixed by #591
